toolbox/siestabarriers/SiestaBarriers.py

```diff
@@ -0,0 +1,245 @@
+                 tolerance_radius =  [1.0,1.0,1.0]
+
+                 ):
+        
+        super().__init__(
+                         host_path ,
+                         host_fdf_name ,
+                         host_structure ,
+                         initial_relaxed_path ,
+                         initial_relaxed_fdf_name ,
+                         initial_structure ,
+                         final_relaxed_path ,
+                         final_relaxed_fdf_name ,
+                         final_structure ,
+                         image_direction ,
+                         trace_atom_initial_position ,
+                         trace_atom_final_position ,
+                         kicked_atom_final_position ,
+                         switched_atom_final_position ,
+                         ring_atoms_paths ,
+                         number_of_images ,
+                         neb_scheme ,
+                         flos_path ,
+                         ghost ,
+                         relaxed ,
+                         flos_file_name_neb,
+                         flos_file_name_relax,
+                         neb_results_path,
+                         relax_engine,
+                         interpolation_method,
+                         tolerance_radius,
+
+                         )
+        
+
+        self.host_path = host_path
+        print(self.host_path)
+        self.host_fdf_name = host_fdf_name
+        self.host_structure = host_structure
+        self.initial_relaxed_path = initial_relaxed_path
+        self.initial_relaxed_fdf_name = initial_relaxed_fdf_name
+        self.initial_structure = initial_structure
+        self.final_relaxed_path = final_relaxed_path
+        self.final_relaxed_fdf_name = final_relaxed_fdf_name
+        self.final_structure = final_structure
+        self.image_direction = image_direction
+        self.trace_atom_initial_position = trace_atom_initial_position
+        self.trace_atom_final_position = trace_atom_final_position
+        self.kicked_atom_final_position = kicked_atom_final_position
+        self.switched_atom_final_position = switched_atom_final_position
+        self.ring_atoms_paths = ring_atoms_paths
+        self.number_of_images = number_of_images
+        print(self.number_of_images)
+        self.neb_scheme = neb_scheme
+        self.flos_file_name_neb = flos_file_name_neb 
+        self.interpolation_method = interpolation_method
+        self.ghost = ghost
+        #self.flos_path = flos_path
+        self.tolerance_radius = tolerance_radius
+       
+        #super(SiestaBarriersBase,self).__init__()
+        #super().__init__()
+        
+        
+
+        print ("---------------------------")
+        print ("Wellcome To SiestaBarriers")
+        print ("      Version : {}".format(__version__))
+        print ("---------------------------")
+        
+        self.setup() 
+
+        if self.neb_scheme == "vacancy-exchange":
```

Comment:
It seems you are "hosting" class differences in a main class, which seems rather strange. Wouldn't it me more natural to have the class describe the intent with proper inheritance?